gateway/dataimpl/server_v1/queryadminserver.go

```diff
@@ -368,6 +374,20 @@ func (s *QueryIndexAdminServer) CreatePrimaryIndex(
 		return nil, s.errorHandler.NewGenericStatus(err).Err()
 	}
 
+	scopeName := s.normalizeDefaultName(in.ScopeName)
+	collectionName := s.normalizeDefaultName(in.CollectionName)
```

Comment:
nit: don't need to assign these to variables, could just put straight in the EnsureQueryIndexCreatedOptions declaration

Reply:
I quite like declaring them just for ease of debugging, but if we want to stick to any specific style then I'm fine with that.